src/Frontend/Components/AttributionColumn/AttributionForm.tsx

```diff
@@ -0,0 +1,118 @@
+// SPDX-FileCopyrightText: Meta Platforms, Inc. and its affiliates
+// SPDX-FileCopyrightText: TNG Technology Consulting GmbH <https://www.tngtech.com>
+//
+// SPDX-License-Identifier: Apache-2.0
+import MuiBox from '@mui/material/Box';
+import MuiDivider from '@mui/material/Divider';
+import MuiToggleButton from '@mui/material/ToggleButton';
+import MuiToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import MuiTypography from '@mui/material/Typography';
+import { ReactElement } from 'react';
+
+import { DisplayPackageInfo } from '../../../shared/shared-types';
+import { text } from '../../../shared/text';
+import { AttributionType } from '../../enums/enums';
+import { setTemporaryDisplayPackageInfo } from '../../state/actions/resource-actions/all-views-simple-actions';
+import { useAppDispatch } from '../../state/hooks';
+import { Confirm } from '../ConfirmationDialog/ConfirmationDialog';
+import { AuditingOptions } from './AuditingOptions';
+import { CommentStack } from './CommentStack';
+import { CopyrightSubPanel } from './CopyrightSubPanel';
+import { LicenseSubPanel } from './LicenseSubPanel';
+import { PackageSubPanel } from './PackageSubPanel';
+
+const classes = {
+  formContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+    padding: '6px',
+    gap: '12px',
+    overflow: 'hidden auto',
+  },
+};
+
+interface AttributionFormProps {
+  packageInfo: DisplayPackageInfo;
+  isEditable: boolean;
+  showHighlight?: boolean;
+  confirmEditWasPreferred?: Confirm;
+}
+
+export function AttributionForm(props: AttributionFormProps): ReactElement {
```

Comment:
no need for a return type of react components (`ReactElement` is also not correct since a React FC can also return null, undefined, etc.)
```suggestion
export function AttributionForm(props: AttributionFormProps) {
```